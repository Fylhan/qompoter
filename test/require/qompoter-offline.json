{
	"name": "require/qompoter-offline",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"git/acme": "dev-master",
		"cp/fylhan": "v1.0"
	},
	"require-dev": {
		"git/acme-tag": "v1.0"
	}
}
