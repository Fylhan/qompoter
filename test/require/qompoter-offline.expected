Qompoter
======== require

* git/acme/dev-master
* cp/fylhan/v1.0
* git/acme-tag/v1.0

done
